src/main/java/com/avispl/symphony/dal/communicator/lg/lcd/LgLCDDevice.java

```diff
@@ -1690,11 +2166,74 @@ private AdvancedControllableProperty createButton(String name, String label, Str
 
 	/**
 	 * This method is used to validate input config management from user
-	 *
-	 * @return boolean is configManagement
 	 */
-	private void isValidConfigManagement() {
+	private void convertConfigManagement() {
 		isConfigManagement = StringUtils.isNotNullOrEmpty(this.configManagement) && this.configManagement.equalsIgnoreCase(LgLCDConstants.IS_VALID_CONFIG_MANAGEMENT);
 	}
-}
 
+	/**
+	 * This method is used to validate input cachingLifetime from user
```

Comment:
this method is used to convert or validate the user input? please edit the java docs or the method name